source/mainnet/net/web3-id/concordia.rst

```diff
@@ -0,0 +1,32 @@
+.. _concordia:
+.. include:: ../../variables.rst
+
+=========
+Concordia
+=========
+
+Concordia is an example solution built on Web3 ID. It allows you to transfer trust between Telegram and Discord. Using verifiable credentials in the |bw|, you can prove ownership of accounts on other platforms. This is done by allowing users to link their accounts and, optionally, real name. You can also ``/check`` and ``/verify`` other users in Telegram and Discord. In other words, if you know John Doe in one platform, you can also trust him in the other platform using his verifiable credentials.
```

Comment:
With Concordia, you can only prove ownership of account on Telegram and Discord, not any other platforms. This is how it works now, maybe in future it will be possible.